src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1370,6 +1364,16 @@ Namespace Microsoft.CodeAnalysis.Operations
             Return New LazyVariableDeclarationGroupOperation(declarations, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundLocalDeclarationOperation(boundLocalDeclaration As BoundLocalDeclaration) As IVariableDeclarationGroupOperation
+            Dim declarations As Lazy(Of ImmutableArray(Of IVariableDeclarationOperation)) =
+                New Lazy(Of ImmutableArray(Of IVariableDeclarationOperation))(Function() GetVariableDeclarationStatementVariables(ImmutableArray.Create(Of BoundLocalDeclarationBase)(boundLocalDeclaration)))
+            Dim syntax As SyntaxNode = boundLocalDeclaration.Syntax
+            Dim type As ITypeSymbol = Nothing
+            Dim constantValue As [Optional](Of Object) = New [Optional](Of Object)()
+            Dim isImplicit As Boolean = boundLocalDeclaration.WasCompilerGenerated
+            Return New LazyVariableDeclarationGroupOperation(declarations, _semanticModel, syntax, type, constantValue, isImplicit)
```

Comment:
>isImplicit [](start = 120, length = 10)

Should this node be always implicit since we are reusing the same syntax for it?

Reply:
I made the binder change in the PR to ensure `BoundLocalDeclaration` generated for this code path is always marked as compiler generated. Are you suggesting asserting that here or in the test visitors?

Reply:
I am suggesting to always pass true, because regardless how we mark BoundLocalDeclaration, this is going to be the second node for the same syntax.

---
In reply to: [198340134](https://github.com/dotnet/roslyn/pull/28135#discussion_r198340134) [](ancestors = 198340134)